R/FilterUnivariateCox.R

```diff
@@ -0,0 +1,101 @@
+#' @title Univariate Cox Survival Filter
+#'
+#' @name mlr_filters_univariatecox
+#'
+#' @description Calculates scores for assessing the relationship between
+#' individual features and the time-to-event outcome (right-censored survival
+#' data) using a univariate Cox proportional hazards model.
+#' The goal is to determine which features have a statistically significant
+#' association with the event of interest, typically in the context of clinical
+#' or biomedical research.
+#'
+#' This filter fits a [CoxPH][mlr3proba::LearnerSurvCoxPH()] learner using each
+#' feature independently and extracts the \eqn{p}-value that quantifies the
+#' significance of the feature's impact on survival. The filter value is
+#' `-log10(p)` where `p` is the \eqn{p}-value. This transformation is necessary
```

Comment:
is this common? Is their a reference why this is done?

Reply:
the log10 is wrong - it's the natural logarithm, I will change it.

Reply:
No particular reference, more like a standard practice and it should be mentioned in some survival textbook, should I search for it?

Reply:
I think it's okay :)